website/content/docs/nia/api/status.mdx

```diff
@@ -225,3 +226,78 @@ Response:
   }
 }
 ```
+
+## Cluster status
+
+The `/v1/status/cluster` API endpoint returns information about high-availability clusters and its members, such as health status and leadership. This endpoint is only supported when using CTS in [high availability mode](/docs/nia/usage/run-ha). If you call the endpoint without configuring CTS for high availability, then CTS prints an error to the console. Refer to [Error Messages](/docs/nia/usage/errors-ref) for information about CTS error messages. 
+
+| Method | Path              | Response format    |
+| ------ | ----------------- | ------------------ |
+| `GET`  | `/status/cluster` | `application/json` |
+
+### Request parameters
+
+Currently no request parameters are offered for the overall status API.
+
+### Request statuses
+
+- `200`: Successfully retrieved the cluster status
+
+### Response fields
+
+The following table describes the responses that the `status/cluster` endpoint can send. 
+
+| Field | Type | Description |
+| --- | ---- | --- |
+| `cluster_name` | string | Identifies the name of the cluster. |
+| `members` | array | Contains an array of [member objects](#member-objects). Each object in the `members` array represents a CTS instance. |
```

Comment:
For consistency should be `list[member]` instead of `array`

Should the `member` part of `list[member]` should link to the member object below

Reply:
Also consider using `list` instead of the word `array`. Array doesn't appear to be used elsewhere

Reply:
```suggestion
| `members` | list[[member](#member-objects)] | Contains a list of [member objects](#member-objects). Each object in the `members` array represents a CTS instance. |
```